docker/jenkins/Dockerfile.centos7-x86_64

```diff
@@ -63,7 +62,8 @@ RUN cd /opt/rstudio-tools/dependencies/common && /bin/bash ./install-common
 # install Qt SDK
 COPY dependencies/linux/install-qt-sdk /tmp/
 RUN mkdir -p /opt/RStudio-QtSDK && \
-    export QT_SDK_DIR=/opt/RStudio-QtSDK/Qt5.10.1 && \
+    export QT_SDK_DIR=/opt/RStudio-QtSDK/Qt5.11.1 && \
+    export QT_QPA_PLATFORM=minimal && \
```

Comment:
Just out of curiosity, why is this necessary? (Hopefully it doesn't affect any of the platform-specific stuff that Qt does, e.g. choosing the appropriate theme for system file dialogs?)

I think that stuff normally happens at runtime but just want to double check that this change doesn't affect that.

Reply:
`QT_QPA_PLATFORM` is purely for install-time; this allows silently running the online-installer without requiring the containers to have any form of X-Windows installed.

Reply:
Got it! Thanks for the clarification.

Reply:
Got it! Thanks for the clarification.